client/app/components/action-form-input.js

```diff
@@ -0,0 +1,23 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class ActionFormInputComponent extends Component {
+  // represents pasForm model
+  @tracked pasForm;
+
+  // attribute on pasForm model, e.g. 'dcpPfchangeincitymap'
+  @tracked landUseActionField;
+
+  // value that field is to be changed to
+  @tracked newVal;
+
+  // input type, e.g. "text" or "number"
+  @tracked fieldType;
+
+  // updates target attribute on key-up event
+  @action
+  updateAttr(attr, newValue) {
+    this.args.pasForm[attr] = newValue;
+  }
```

Comment:
Ah, I think I see the problem... `args` are immutable, so you have to have this action here that updates the value. Interesting.